pyproject.toml

```diff
@@ -17,13 +17,14 @@ authors = [
 
 [tool.poetry.dependencies]
 python = "^3.9"
-python-telegram-bot="13.0"
-pymongo="^3.11.0"
-google-cloud-translate="2.0.0"
-google-cloud-automl="0.10.0"
-google-cloud-speech="2.0.0"
-sentry-sdk="^0.19"
-Pillow="^8.0"
+python-telegram-bot = "13.0"
+pymongo = "^3.11.0"
+google-cloud-translate = "2.0.0"
+google-cloud-automl = "0.10.0"
+google-cloud-speech = "2.0.0"
+sentry-sdk = "^0.19"
+Pillow = "^8.0"
+toml = "0.10.2"
```

Comment:
You have added just one dependency but changed formatting for the whole section. Don't do it. 🙅

Reply:
Fixed.